src/popupmenu.c

```diff
@@ -485,7 +485,7 @@ pum_screen_put_with_attr(int row, int col, char_u *text, int textlen, hlf_T hlf)
 	    }
 	}
 	else if (!in_fuzzy && (ptr - text < leader_len)
-			     && (STRNCMP(text, match_leader, leader_len) == 0))
+			     && (MB_STRNICMP(text, match_leader, leader_len) == 0))
```

Comment:
This function is called repeatedly when it only needs to be called once. Maybe you can rebase this onto #15020.

Reply:
oh nice. i will wait that.

Reply:
mark as draft then.

Reply:
I have merged it.